### ISSUE TITLE
Aquisição de+ de 1 curso do site

### ISSUE THREAD

**Issue by tandyguit** [san0suke/pagseguro-assinaturas-rcs#26]
Olá.
Seu plugin é excelente, adorei, está ajudando bastante. Mas queria tirar uma dúvida:
Quando o cliente adquire 2 cursos ele não consegue enviar para o carrinho de compras, todas vez que é lançado uma comprar o carrinho atualiza para limpar e inserir a nova aquisição.
Pra mim está dificultando a compra do cliente no site, pois ele não consegue inserir 2 cursos no carrinho.
Como posso corrigir isso no arquivo que criou do plugin?

**Comment by tandyguit** [comment]
Na verdade gostaria que não fosse bloqueado adicionar mais de um produto. Vendo livros e quando o cliente adiciona mais de 1 livro, o anterior escolhido é apagado pelo plugin e adicionado o novo. E isso está me atrapalhando.

**Comment by san0suke** [comment]
O que daria pra fazer é uma única assinatura com os dois preços somados. Não tem como duas ao mesmo tempo por conta do PagSeguro.

**Comment by tandyguit** [comment]
Como posso liberar essa situação na programação. Estou tendo problema na compra dos materiais, pois o cliente não pode adicionar +1 de produto por vez no carrinho, toda vez que 
é adicionado o 2o produto o carrinho é limpo e permanece somente 1 sendo o mais recente.

**Comment by san0suke** [comment]
O plugin só bloqueia compra de múltiplos itens ao mesmo tempo se forem assinaturas.

**Comment by tandyguit** [comment]
No caso do meu site, ele está apagando qualquer um. Como faço pra corrigir isso?

**Comment by san0suke** [comment]
Acho que você não está entendendo. Você não consegue ter uma assinatura e um produto não assinatura no carrinho ao mesmo tempo. Sugiro que você faça duas seções no seu site, uma só produtos e outra só assinaturas.

**Comment by tandyguit** [comment]
Sim entendi. Mas veja, mesmo que seja 2 livros no mesmo carrinho não estando nenhuma assinatura nele, seu plugin não deixa entrar os dois livros ou seja a programação feita no seu plugin barra ter +de 1 produto por compra mesmo não sendo assinatura. Com relação a serviços de assinaturas está ok, conforme programou, mas está me atrapalhando caso o cliente queira comprar 5 livros de uma vez só por exemplo onde o serviço ativado é o pagseguro normal.

**Comment by san0suke** [comment]
Verifique se as outras formas de pagamento estão configuradas.

**Comment by san0suke** [comment]
Existe também uma opção que habilita outras formas de pagamento quando o meu plugin está ativado. Veja se está opção está marcada.
Se o filtro estiver atrapalhando, solicite a um desenvolvedor que altere a classe PGA_Compra.class. Toda a lógica que causa isso está nessa classe.

**Comment by tandyguit** [comment]
Sim. Vc fez uma opção que quando for plano de assinatura, o sistema só ativar o pagamento por recorrência, bloqueando outras formas de pagamentos.
Teste só no meu site www.matematicapassoapasso.com.br
